### DISCUSSION TITLE
Treating Handlebars as HTML files

### DISCUSSION THREAD

**Discussion by frangio** [parcel-bundler/parcel#5260] [Q&A] [answered]
I have a set of handlebars templates (`*.hbs`) that I would like to bundle exactly as if they were HTML files. The only difference is the extension. How can I achieve this? I tried the following configuration but the `*.hbs` file is just copied: none of the assets it references are being loaded.

```
{
  "extends": ["@parcel/config-default"],
  "transformers": {
    "*.hbs": ["@parcel/transformer-posthtml"]
  },
  "packagers": {
    "*.hbs": "@parcel/packager-html"
  }
}
```

Thank you.

**Comment by mischnic** [comment, marked as answer]
This should work
```
{
  "extends": ["@parcel/config-default"],
  "transformers": {
    "*.hbs": ["@parcel/transformer-posthtml", "@parcel/transformer-html"]
  }
}
```

**Reply by frangio** [reply]
Awesome! That almost works. The `.hbs` file is emitted as an `.html` file. Is it possible to keep the extension?

**Reply by mischnic** [reply]
Create an optimizer that sets `bundle.type = 'hbs';`. But this might cause a problem if you also have other HTML bundles that should actually stay html bundles
